src/tables/sortable-table.js

```diff
@@ -92,101 +100,123 @@ function SortableTable({
-  const [sortFunc, setSortFunc] = useState(() => initialSortFunc)
-  const [valueGetter, setValueGetter] = useState(() => initialValueGetter)
+  const [sortFunc, setSortFunc] = useState(() => initialSortFunc);
+  const [valueGetter, setValueGetter] = useState(() => initialValueGetter);
 
   const data = useMemo(() => {
-    if (controlled || disableSort) return unsortedData
+    if (controlled || disableSort) return unsortedData;
 
     if (sortFunc) {
-      const sorted = [...unsortedData].sort(sortFunc)
-      if (!ascending && !disableReverse) sorted.reverse()
-      return sorted
-    }
-    else {
-      const order = ascending ? 'asc' : 'desc'
+      const sorted = [...unsortedData].sort(sortFunc);
+      if (!ascending && !disableReverse) sorted.reverse();
+      return sorted;
+    } else {
+      const order = ascending ? "asc" : "desc";
       const sorted = orderBy(
         unsortedData,
-        (item) => valueGetter ? valueGetter(item) : get(sortPath, item),
+        (item) => (valueGetter ? valueGetter(item) : get(sortPath, item)),
         order
-      )
-      return sorted
+      );
+      return sorted;
     }
-  }, [ascending, sortPath, sortFunc, valueGetter, controlled, disableSort, disableReverse, unsortedData])
+  }, [
+    ascending,
+    sortPath,
+    sortFunc,
+    valueGetter,
+    controlled,
+    disableSort,
+    disableReverse,
+    unsortedData,
+  ]);
 
   const handleColumnChange = (column) => {
-    if (column.disabled) return
+    if (column.disabled) return;
 
-    const newSortPath = column.name
-    const newSortFunc = column.sortFunc || null
-    const newValueGetter = column.valueGetter || null
+    const newSortPath = column.name;
+    const newSortFunc = column.sortFunc || null;
+    const newValueGetter = column.valueGetter || null;
 
     // Toggle ascending if the path is already selected. Otherwise, default
     // to ascending when switching paths...
-    const newAscending = newSortPath === sortPath ? !ascending : true
+    const newAscending = newSortPath === sortPath ? !ascending : true;
 
-    setAscending(newAscending)
-    setSortPath(newSortPath)
-    setSortFunc(() => newSortFunc)
-    setValueGetter(() => newValueGetter)
+    setAscending(newAscending);
+    setSortPath(newSortPath);
+    setSortFunc(() => newSortFunc);
+    setValueGetter(() => newValueGetter);
 
-    if (onChange) onChange({
-      ascending: newAscending,
-      sortPath: newSortPath,
-      sortFunc: newSortFunc
-    })
-  }
+    if (onChange)
+      onChange({
+        ascending: newAscending,
+        sortPath: newSortPath,
+        sortFunc: newSortFunc,
+      });
+  };
 
   return (
-    <table className={classnames(className, { 'sortable-table': !disableSort })} {...filterInvalidDOMProps(rest)}>
-      <thead>
-        <tr>
-          {columns.map((column, key) => {
-              const Header = column.headerComponent || headerComponent || DefaultHeader
+    <>
+      <CaptionComponent caption={caption} />
```

Comment:
As a `node`, you could just always render the caption. E.g.,
```jsx
<caption>{caption}</caption>
```
However, since we'd be introducing a new change this could impact styling on existing projects. I believe this is tagged to the `v6` milestone, so please modify the base branch to `v6` and add a note in the [migration guide](https://github.com/LaunchPadLab/lp-components/blob/7da7efea655a0716e5988eb676be68472f1d5106/migration-guides/v6.0.0.md)